h2o-test-support/src/main/java/water/fvec/TestFrameBuilderTest.java

```diff
@@ -253,5 +253,21 @@ public void testBadVec(){
       Scope.exit();
     }
   }
+
+  @Test
+  public void testSequenceIntDataForCol() {
+    Scope.enter();
+    int numRows = 10;
+    try {
+      Frame f = new TestFrameBuilder()
+              .withVecTypes(Vec.T_NUM)
+              .withSequenceIntDataForCol(0, 0, numRows)
+              .build();
+      Scope.track(f);
+      assertVecEquals(Vec.makeSeq(0, numRows), f.vec(0), 0);
```

Comment:
this will only be true for low number of rows because it will create a single chunk, generally it is not true